public/plugins/testdata/tsconfig.json

```diff
@@ -3,5 +3,6 @@
   "compilerOptions": {
     "typeRoots": []
   },
-  "include": ["./src", "./types", "../../app/types/*.d.ts", "../../../packages/grafana-ui/src/types/*.d.ts"]
+  "include": ["./src", ".config/types"],
+  "exclude": ["**/*.test.ts", "**/*.test.tsx", "**/*.spec.ts", "**/*.spec.tsx"]
```

Comment:
this was necessary because test files reported missing definitions like:

```
src/metricTree.test.ts:22:5 - error TS2304: Cannot find name 'expect'.

22     expect(nodes).toEqual(expect.arrayContaining(['ABA', 'ABB', 'ABC']));
```